### PR TITLE
fix: canceled order mail template changed

### LINKED ISSUE #6642: Canceled orders with invoice links to 404 page
If an organizer cancels a ticket the user receives an email with "Click here to view/download the invoice." and a wrong link. The entire message is:

> Hi,Your order for TechSummit 2020 has been cancelled has been cancelled by the organizer
Please contact the organizer for more info
Message from the organizer: None
Click here to view/download the invoice.
Login to manage the orders at https://eventyay.com

Screenshot from 2019-12-06 18-44-40

Please change the email as follows. Include punctuation and line breaks as follows.

Hello,

your order for [name of event] has been cancelled by the organizer. 

Please contact the organizer for more info. 

To manage orders please login to https://eventyay.com and visit "My Tickets".

Best regards,

Eventyay Team

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6642 

#### Short description of what this resolves:
fix: canceled order mail template changed

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@prateekj117 @iamareebjamal Review if possible!

File: app/api/helpers/system_mails.py [hunk 1]
```diff
@@ -186,11 +186,12 @@
...
-            u"<br/>Please contact the organizer for more info" +
-            u"<br/>Message from the organizer: {cancel_note}"
-            u"<br/> <a href='{order_url}'>Click here</a> to view/download the invoice."
-            u"<br/>Login to manage the orders at {frontend_url} </em>"
+            u"Hello,"
+            u"<br/>your order for {event_name} has been cancelled by the organizer."
+            u"<br/>Please contact the organizer for more info." +
+            u"<br/>To manage orders please login to {frontend_url} and visit \"My Tickets\"."
+            u"<br/>Best regards,"
+            u"<br/>Eventyay Team"
```
Comment (iamareebjamal) [inline app/api/helpers/system_mails.py]:
```suggestion
            u"<br/>Eventyay Team"
```

See the repository name

File: app/api/helpers/mail.py [hunk 2]
```diff
@@ -369,12 +369,9 @@ def send_order_cancel_email(order):
         subject=MAILS[TICKET_CANCELLED]['subject'].format(
             event_name=order.event.name,
             invoice_id=order.invoice_number,
-            frontend_url=get_settings()['frontend_url']
```
Comment (kushthedude) [inline app/api/helpers/mail.py]: Why

Comment (codedsun) [reply app/api/helpers/mail.py]: How is this field utilized in the subject, I didn't see it used anywhere in the text

**Comment by codedsun** [comment]
Review this!

File: app/api/helpers/mail.py (hunk 2 above)
Comment (iamareebjamal) [reply app/api/helpers/mail.py]: I do

File: app/api/helpers/mail.py [hunk 3]
```diff
@@ -363,18 +363,21 @@ def send_email_to_attendees(order, purchaser_id, attachments=None):
 
 
 def send_order_cancel_email(order):
+    cancel_msg = ''
+    if order.cancel_note:
+        cancel_msg = u"<br/>Message from the organizer: {cancel_note}".format(cancel_note=order.cancel_note)
```
Comment (iamareebjamal) [inline app/api/helpers/mail.py]: No

File: app/api/helpers/mail.py (hunk 2 above)
Comment (codedsun) [reply app/api/helpers/mail.py]: I got it!

File: app/api/helpers/mail.py (hunk 3 above)
Comment (codedsun) [reply app/api/helpers/mail.py]: if the cancel_note is not none then Message from the organizer should go in the email or not? I tried this way

Comment (codedsun) [reply app/api/helpers/mail.py]: Is the way of doing this wrong or something else you wanted to say?

Comment (iamareebjamal) [reply app/api/helpers/mail.py]: OK for now

**Comment by codedsun** [comment]
I have updated the branch, if it's okay. Please merge. Thanks

**Pull request merged**